src/Symfony/Bundle/WebProfilerBundle/Resources/config/routing/profiler.xml

```diff
@@ -36,6 +36,10 @@
         <default key="_controller">web_profiler.controller.profiler:panelAction</default>
     </route>
 
+    <route id="_profiler_replay" path="/{token}/replay">
```

Comment:
a route accessible in GET but allowing to replay non-safe requests is breaking the HTTP rules, as your GET endpoint becomes unsafe.